anomalib/deploy/inference.py

```diff
@@ -21,7 +21,7 @@
 import cv2
```

Comment:
Can we rename this file to `inferencer.py`, to better make the distinction between `tools/inference.py`?